File: tamper-scripts/perl/base64encode.pl
#!/usr/bin/env perl
#
# Author:       KING SABRI | @KINGSABRI
# Description:  Base64 encoding all characters in a given payload
# Requirements: sudo cpan JSON::Parse
#
use strict;
use JSON;
use JSON::Parse ':all';
use MIME::Base64;

my %json    = decode_json $ARGV[0];
my $payload = $json{"payload"};
my $kwargs  = $json{"headers"};

print encode_json \%json

File: tamper-scripts/php/base64encode.php
#!/usr/bin/env php
<?php
#
# Author:       KING SABRI | @KINGSABRI
# Description:  A PHP Template to build on your tamper, copy and build on me.
# Requirements: None
#
$json    = json_decode($argv[1], true);
$payload = $json['payload'];
$kwargs  = $json['headers'];

$json['payload'] = base64_encode($payload);

echo json_encode($json);
?>
